daml-assistant/daml-helper/src/DA/Daml/Helper/Start.hs

```diff
@@ -228,26 +228,29 @@ runStart
     scriptOpts <- withOptsFromProjectConfig "script-options" scriptOpts projectConfig
     doBuild
     doCodegen projectConfig
-    listenForKeyPress projectConfig darPath
     let scenarioArgs = maybe [] (\scenario -> ["--scenario", scenario]) mbScenario
     withSandbox sandboxClassic sandboxPort (darPath : scenarioArgs ++ sandboxOpts) $ \sandboxPh sandboxPort -> do
+        let doRunInitScript =
+              whenJust mbInitScript $ \initScript -> do
+                  putStrLn "Running the initialization script."
+                  procScript <- toAssistantCommand $
+                      [ "script"
+                      , "--dar"
+                      , darPath
+                      , "--script-name"
+                      , initScript
+                      , if any (`elem` ["-s", "--static-time"]) sandboxOpts
+                          then "--static-time"
+                          else "--wall-clock-time"
+                      , "--ledger-host"
+                      , "localhost"
+                      , "--ledger-port"
+                      , case sandboxPort of SandboxPort port -> show port
+                      ] ++ scriptOpts
+                  runProcess_ procScript
```

Comment:
This is getting more and more complex. Do you think there's a chance we could have an integration test that the `R` keypress works as intended?

Reply:
Yes we should probably add one. Would you be ok to do this in a follow-up PR as I think that test will be a bit more involved?

Reply:
I merge this one for now and start working on an integration test.

Reply:
Adding this in a separate PR would have been my suggestion as well. Thanks for taking care of it.